WakeLockSentinel/WakeLockSentinel_onrelease_EventHandler.html

```diff
@@ -0,0 +1,49 @@
+<div>{{securecontext_header}}{{APIRef("Screen Wake Lock API")}}</div>
```

Comment:
I think my templating has messed this up. There should be `onrelease`, the write property that takes the even callback and `Release` the event passed to the callback. 

ALL of the text you've written is usable in some form. I'll fix this since it's my fault. The next time you do an API with an even handler ping me to make sure the `build` command spits out the right content. (https://github.com/jpmedley/mdn-helper/issues/49)

Reply:
The content index docs I think were generated correctly. I had the option when I generated these docs for both ContentIndex _and_ ContentIndexEvent (which I have done). The event doesn't come up with a use the helper for wake lock - possibly could be a mismatch of info somewhere.

Do you have a good example of event interface pages? Something for inspiration? :)